_chapters/chapter_11.md

```diff
@@ -281,7 +290,7 @@ layout: rails_tutorial
 
   1.  We're rendering a blank page for the new reviews page, but what should we be rendering instead?
 
-      We're going to follow the pattern we used before to add books.
+      We're going to follow the pattern we used before to add books back in chapter 5 of this tutorial.
```

Comment:
Maybe this is a little better?

"We're going to follow the same we pattern we used back in Chapter 5 to add books."